src/components/VideoPlayer.vue

```diff
@@ -38,6 +38,12 @@ export default {
                 return {};
             },
         },
+        nextVideo: {
+            type: Object,
+            default: () => {
+                return {};
+            },
+        },
```

Comment:
Shouldn't this be a String?

Reply:
I was thinking it should be a video object because we might display what's playing next when hovering on the Next button like youtube does. So for this we would need title and thumbnail as well

Reply:
Okay, fair enough. We could potentially move the autoplay selection logic entirely to `WatchVideo.vue` in that case!